Calendar/Calendar/CalendarForm.cs

```diff
@@ -128,12 +161,20 @@ private void OnDayDoubleClicked(object sender, EventArgs e)
                 }
             }
         }
-
+        /// <summary>
+        /// The function NextMonth() starts after clicking the nextButton.
```

Comment:
tutaj np komentarz jest zbędny. To jest tylko handler który woła metodę nic więcej. Ważne żeby ta metoda miałą komentarz. Jakby było np 5 metod wołanych to bym opisał co ten handler OGÓLNIE robi. Np STARTS PROCESSING OF USER ORDER (jeśli to byłby sklep internetowy i ktoś kliknąłby DO KASY). Ale tutaj logika jest prosta.